README.md

```diff
@@ -38,3 +38,45 @@ SiteSetting.sso_overrides_name = true
 SiteSetting.sso_overrides_email= true
 ```
 
+## Import from v5
+
+Loading the v5 database
```

Comment:
Perhaps you could add the triple backquotes to make the command lines appear as code?